### PR TITLE
#182: Add button-platforme

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | [#182](https://github.com/ripe-tech/ripe-white/issues/182) |
| Decisions | Button Platforme needed for the Ripe ID Tech plugin |
| Animated GIF | -- |

### CODE REVIEW

File: vue/components/atoms/button-platforme/button-platforme.vue [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+.button.button-platforme > span.button-logo {
+    background-color: #151515;
+    float: left;
+    height: 44px;
+    line-height: 44px;
+    margin-right: 10px;
+    text-align: right;
+    // transition: background-color 0.15s ease-in-out;
```
Comment (gcandal) [inline vue/components/atoms/button-platforme/button-platforme.vue]: oops?

File: vue/components/atoms/button-platforme/button-platforme.vue [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    width: 44px;
+}
+
+.button.button-platforme.loading > span.button-logo {
+    opacity: 0.7;
+}
+
+// .button.button-platforme:hover > span.button-logo {
```
Comment (gcandal) [inline vue/components/atoms/button-platforme/button-platforme.vue]: oops?

File: js/base/main.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import "ripe-sdk/src/css/ripe.css";
 
 import { Ripe } from "ripe-sdk";
+import { API as RipeIdAPI } from "ripe-id-api";
```
Comment (gcandal) [inline js/base/main.js]: @BeeMargarida let's keep this scoped to where it's being used: the technical plugin. Make this PR only about adding the button to commons.

Comment (BeeMargarida) [reply js/base/main.js]: @gcandal So I only initialize this in the ripe id plugin?

Comment (gcandal) [reply js/base/main.js]: yes :)

**Comment by joamag** [comment]
@gcandal please add a story to this component and a simple GIF

**Comment by gcandal** [comment]
@BeeMargarida please add a story and since you're at it do the same for this component in react-components-vue.

**Pull request merged**